js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -1,6 +1,6 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var ExtensionStore = require('./ExtensionStore.js');
+var ExtensionStore = require('./ExtensionStore.js').instance;
```

Comment:
Well, I am morally opposed to statics in almost all cases. I need to have a particular instance configured by the user (e.g. blueocean-web), though, so we don't have another mechanism to do this. I think we should investigate a DI system. The tight coupling of some things has me already feeling worried about the future of this codebase...